posts/2024-05-02-comparison-of-performance.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    def lin_interp(x_0: float, y_0: float, x_1: float, y_1: float, x: float) -&gt; float:
</pre><p>just a minor thing but could you expand the function rather than put all oft his in the 'return'</p><p>like</p><p>interp = y_0 .. and also possibly I would swap y_0 for y0 etc it makes a bit more readable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/coecms/coecms.github.io/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>